README.md

```diff
@@ -9,25 +9,29 @@ The following tutorial guides you through the process of building and running th
 
 # Table of Contents
 
-[Prequisites: Prepare your Environment](#Prequisites-Prepare-your-Environment)
-
-[Build Demo VHD or VHDX Image](#build-demo-vhd-or-vhdx)
-
-[Build Demo ISO Image](#build-demo-iso)
-
-[Image config file](#image-config-file)
-
-[Customize Demo Image with Pre-built Packages](#customize-demo-image-with-pre-built-packages)
-
-[Customize Demo Image with New Packages](#customize-demo-image-with-new-packages)
-
-[Modify the Demo Image Kernel](#modify-the-demo-image-kernel)
-
-# **Prequisites: Prepare your Environment**
+- [Prequisites: Prepare your Environment](#prequisites-prepare-your-environment)
+  - [Install Tools](#install-tools)

```

Comment:
Please restore the TOC with just the "major" headings.  Trying to keep this lighter.

Reply:
Will do. I've extended it because it was hard to find the new instructions without it. Would you like me to move them to its own chapter and add only one entry to the TOC?